### PR TITLE
Simple ObjectModel performance improvement

### PR DESCRIPTION
**Pull request by firstred**
| Questions | Answers |
| --- | --- |
| Branch? | `develop`, `1.6.1.x` |
| Description? | A simple performance improvement for ObjectModel, by decreasing the amount of calls to `Configuration::get` |
| Type? | improvement |
| Category? | CO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/PSCSX-2548 |
| How to test? |  |

### CODE REVIEW

File: classes/ObjectModel.php [hunk 1]
```diff
@@ -923,13 +923,14 @@ public function validateFields($die = true, $error_return = false)
...
-     * @param bool $error_return
+     * @param bool $errorReturn
      *
      * @return bool|string True, false or error message.
      * @throws PrestaShopException
      */
-    public function validateFieldsLang($die = true, $error_return = false)
+    public function validateFieldsLang($die = true, $errorReturn = false)
     {
+        $defaultLang = (int) Configuration::get('PS_LANG_DEFAULT');
```
Comment (unlocomqx) [inline classes/ObjectModel.php]: I'd like the point out the space after `(int)`, it makes me uncomfortable :laughing:

Comment (firstred) [inline classes/ObjectModel.php]: Sorry, that's the new standard now.

Comment (unlocomqx) [inline classes/ObjectModel.php]: I didn't check the standard but I checked the whole file for other occurrences, I'll get used to it then, cheers

**Comment by aleeks** [comment]
Thank you @firstred !

**Comment by xBorderie** [comment]
Is there a 1.6 version of this PR?

**Comment by kpodemski** [comment]
@firstred just curious, did you check profiler if this is even worth this change? because entire Configuration is cached

**Pull request merged**